newjersey/navigator.business.nj.gov
# feat: remove-account-creation-card-#185681589

## Description



### Ticket



[#185681589](https://www.pivotaltracker.com/story/show/185681589)

### Approach



### Steps to Test


When you are prompted to created an account, close the **Link with myNewJersey to Save Your Progress** snack bar. A create an account card should appear. Once you create your account, the card confirmation that once appeared in the **For You*** section, will no longer appear.

### Notes



## Code author checklist
